coralproject/talk
# Futureproof Embed Code

Story: https://www.pivotaltracker.com/story/show/138858105
Implementation Plan: https://github.com/gobengo/talk/wiki/Embed-Code-for-Talk-Stream

New embed code like:
```
<script src="http://your-talk-installation/Coral.js" onload="
      Coral.Talk.render(document.getElementById('some-div'), {
        talk: 'http://your-talk-installation'
      })
"></script>
```

## What does this PR do?

* Add client/coral-embed-api module. It's meant to be included in an HTML page and define window.Coral
* Serve coral-embed-api at /Coral.js
* Update / handler (article view) to use new embed format
* Update Talk Admin embed code section to use new embed format

## How do I test this PR?

* The embed at '/' should load normally
* In the Admin -> Configure -> Embed Code pane, you should see a new format (containing 'Coral.Talk.render...'). And you should be able to copy paste that into a jsbin and see the embed load
